app/src/main/java/space/taran/arknavigator/utils/DialogUtils.kt

```diff
@@ -0,0 +1,51 @@
+package space.taran.arknavigator.utils
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import androidx.appcompat.app.AlertDialog
+import androidx.core.content.ContextCompat
+import space.taran.arknavigator.R
+import space.taran.arknavigator.databinding.DialogInfoBinding
+
+@SuppressLint("ClickableViewAccessibility")
+fun showInfoDialog(
+    context: Context,
+    @StringRes title: Int,
+    @StringRes descrText: Int,
+    @StringRes posButtonText: Int = R.string.ok,
+    @StringRes negButtonText: Int? = null,
+    posButtonCallback: (() -> Unit)? = null,
+    negButtonCallback: (() -> Unit)? = null
+) {
+    val builder = AlertDialog.Builder(context, R.style.AppTheme_AlertDialogStyle)
+    val view = View.inflate(context, R.layout.dialog_info, null)
+    val binding = DialogInfoBinding.bind(view)
```

Comment:
We should use the FragmentDialog and Fragment Result API. You can use the `EditTagsDialogFragment`(PR #135) as sample.